### ISSUE TITLE
how can we access jwt token

### ISSUE THREAD

**Issue by masoudei** [sidebase/nuxt-auth#86]
### Ask your question

how can we access jwt token in our Nuxt composable as fetch interceptor .. 
no way to access the original JWT token we received from Credentials provider ?

**Comment by BracketJohn** [comment]
Hey @masoudei!

To access the JWT token you can use `getToken` on the server side, as documented here:
- old docs: https://github.com/sidebase/nuxt-auth#getting-the-jwt-token
- new docs: sidebase.io/nuxt-auth/server-side/jwt-access (once #90 is merged)

If you want to access the JWT token when it is being created, add the `jwt` callback to your `NuxtAuthHandler`. You can see the full docs for that here: https://next-auth.js.org/configuration/callbacks#jwt-callback

Does this answer your question?

**Comment by BracketJohn** [comment]
closing this now, as question was answered and no more response.

Let me know here if this persists or open a new issue if you have a new question!
